### DISCUSSION TITLE
timestamp

### DISCUSSION THREAD

**Discussion by sfraysse** [gaia-x-dases/xapi-virtual-classroom#8] [General]
I would suggest to make the `timestamp` mandatory.
xAPI does not requires this, but common profiles like CMI5 or video do.

A reason is that the virtual classroom platform may send the statements to the LRS asynchronously (with a certain delay).
In this case, the `timestamp` should be the time of the event, while the `stored` prop will be the time of recording by the LRS.

**Comment by quitterie-lcs** [comment]
Indeed, you're right
